tests/integration/test_remoted/test_configuration/test_basic_configuration_queue_size_valid.py

```diff
@@ -42,8 +42,12 @@ def get_configuration(request):
 
 def test_queue_size_valid(get_configuration, configure_environment, restart_remoted):
     """
+    Checks that `queue_size` option could be configured with valid values (any number between 1 and 262144)
+    without errors
 
+    Checks that the API answer for manager connection coincides with the option selected on ossec.conf
```

Comment:
I prefer to give a brief description of the test in a few words. If you need to add more details, I prefer to add a new section below giving all the details you need. For example:
```
"""Brief description of the test

Lore ipsum....

Raises:
     Exception
```